tests/extractors/cookie_test.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+import pytest
+
+from app.splash_models import HAR
+from features.cookies import Cookies
+from tests.integration.features_integration_test import mock_website_data
+
+
+@pytest.mark.asyncio
+async def test_cookies_har(executor):
+    feature = Cookies()
+    await feature.setup()
+
+    har = HAR.parse_obj(
+        json.loads(
+            """{
+        "log": {
+            "entries": [
+                {
+                    "response": {
+                        "headers": [],
+                        "cookies": [
+                            {
+                                "name": "test_response_cookie",
+                                "value": "dummy",
+                                "httpOnly": "false",
+                                "secure": "true"
+                            }
+                        ]
+                    },
+                    "request": {
+                        "headers": [],
+                        "cookies": [
+                            {
+                                "name": "test_request_cookie",
+                                "value": "dummy",
+                                "httpOnly": "false",
+                                "secure": "true"
+                            }
+                        ]
+                    }
+                }
+            ]
+        }
+    }"""
+        )
+    )
+    site = await mock_website_data(har=har)
+    stars, explanation, values = await feature.extract(site, executor=executor)
+
+    # extra data returns "cookie-name=cookie-value" pairs for cookies extracted from har
+    assert values == ["test_response_cookie=dummy", "test_request_cookie=dummy"]
+
+
+@pytest.mark.asyncio
+async def test_cookies_html(executor):
+    feature = Cookies()
+    await feature.setup()
+    html = """
+           <div class='ast-small-footer-section ast-small-footer-section-1 ast-small-footer-section-equally ast-col-md-6 ast-col-xs-12' >
```

Comment:
Alongside with reducing the size of code: I guess you could throw out most of this text but still keep the extractor test valid, right?

Reply:
yes, I did that for some, but for some tests I am not sure why they do what they do. This is actually one of them. I disabled the whole test in the follow up PR because I don't know why it is doing what it is doing...

Reply:
:D Ok, so basically the solution to not-understanding is disabling? This cannot be the way to go! Like the other html based extractors, the html is scanned for snippets of code in the tag lists. These snippets are known to be connected to cookies. I constructed an html which contains such a snippet, in this case "iubenda"-something-something. If you have more questions regarding this and other unclear tests let me know. Deactivating tests is not an option, unless of course you replace them with another, equally suitable test.

Reply:
We can move the discussion about that specific test to the follow up PR i guess!

Reply:
Yes, please do so :)